tests/resolve-blank-lines-test/Makefile

```
@@ -1,5 +1,7 @@
 PROGRAM=../../resolve-blank-lines
 
+test: diff
```

checkmake (0.2.2)

[warning] Target "test" should be declared PHONY.

(phonydeclared)